### PR TITLE
Improved readme.md

### PR DESCRIPTION
**Pull request by guillermoojeda**
Simple improves on readme information and readability.

### CODE REVIEW

**Review by natilou — APPROVED** [review]
LGTM

File: readme.md [hunk 1]
```diff
@@ -2,34 +2,46 @@
 
 Proyecto de EDA6 - Legendary Wallabies
 
-## Packages installed:
-To be completed soon.
+Packages installed:\
+completar
```
Comment (natilou) [inline readme.md:6]: we can add here the `Pipfile` dependencies:
```
[packages]
numpy = "1.24.2"
openpyxl = "3.1.2"
pandas = "2.0.0"
dash = "2.9.3"
dash-bootstrap-components = "1.4.1"


[dev-packages]
freezegun = "1.2.2"
ipdb = "0.13.13"
pandas = "2.0.0"
openpyxl = "3.1.2"
pytest = "7.2.2"
flake8 = "6.0.0"

[requires]
python_version = "3.10"
```

Comment (matiasbertani) [reply readme.md]: I refactor a little bit to link the Pifile in main with packages, I don't know how to dynamically update readme with another file information, and think it is not possible.

Let me know if this approach works

Comment (natilou) [reply readme.md]: I think this is a good aproach now. 
Maybe we can create a script to check updates in Pipfile and make the modifications in the readme file. And run this script with github actions. I will investigate it, but for now this approach is very good!

**Review by natilou — APPROVED** [review]
LGTM

**Pull request merged**